e2e/fixtures/global-setup.js

```diff
@@ -0,0 +1,50 @@
+// e2e/fixtures/global-setup.js
+import { test as setup, expect } from '@playwright/test';
+import { getScenarioName } from '../../helpers';
+
+async function createProviderFixture(browser, testInfo) {
+    const page = await browser.newPage();
+    const scenarioName = getScenarioName(testInfo.file);
+    return { page, scenarioName };
+}
+
+async function createCaseworkerFixture(browser, testInfo) {
+    const scenarioName = getScenarioName(testInfo.file);
+    if (!scenarioName) {
+        throw new Error(`Could not determine scenario name from path: ${testInfo.file}`);
+    }
+
+    const storagePath = `./e2e/storage/${scenarioName}-provider-state.json`;
+
+    const context = await browser.newContext({
+        storageState: storagePath
+    });
+    const page = await context.newPage();
+
+    const storageState = await context.storageState();
```

Comment:
Do we have to take any considerations here regarding clearing out this storage? How often is it cleared out?

Reply:
This removes after every test run as we build the e2e before each run. storage folder is temporary and not part of the repo.

Reply:
Not part of the repo no but we should take care with over-using it and having to have branches for what the storage state is in the tests.

Reply:
@elken and @imtiazAhmd agreed - you need to be very careful about how you use the LAAReference numbers here. Could you get rid of it once you have run the provider and associated caseworker test rather than after all tests have run? 
Otherwise the next person to create a journey might think they can use your LAA ref for a different test case and find that the claim/application is not in the state they expect. Or does the use of scenarioName mean it only gets used for the one scenario?

Reply:
The storage is cleared after every test I think Imtiaz said so it should be

Reply:
So it feels to me like the reference numbers are really a symptom of the bigger problem, which is that instead of end-to-end tests we now have a bunch of part-of-journey tests that only work if they are run in a specific order, and we're using a DB persistence and effectively a reusable `laaReference` global variable to join them up. I think @jogilder's concern is absolutely on the money here - because we're not doing each test in the suite from a blank state, and have introduced the pattern of making assumptions about what tests have previously run in subsequent tests, we're opening ourselves up to the next person to overwrite the laa reference in a new test, or accidentally put the new test in the wrong place in the brittle running order, abruptly breaking every other test.

Reply:
Hmm, yes I think you're right there.

This will need further changes to factor out the global state. I wasn't sure in the first place but I assumed things like this were considered before the introduction of it.

Reply:
As an incremental improvement, can I suggest that instead of every pair of scenarios storing their LAA reference in the same place, we make it so that the different LAA references are keyed under different names? That way, even though we're still effectively using global variables, we're not _reusing_ global variables, and there's much less that can go wrong if the order of tests changes.

Reply:
Oh ignore me, I misread your comment above.